scripts/semconv/templates/registry/stable/docstring.ts.j2

```diff
@@ -20,7 +20,7 @@
 {% endif -%}
 
 {%- if obj is deprecated %}
-  {%- set deprecated_jsdoc = "\n@deprecated " ~ strong_rfc2119(obj.deprecated) -%}
+  {%- set deprecated_jsdoc = "\n@deprecated " ~ strong_rfc2119(obj.deprecated.note) -%}
```

Comment:
Reviewer note: The weaver update (from 0.12 to 0.13) changed `obj.deprecated` from possible string to an object with `note` and `reason` properties.